pinns/src/sysctl.c

```diff
@@ -0,0 +1,105 @@
+#define _GNU_SOURCE
+
+#include <string.h>
+#include <fcntl.h>
+
+#include "sysctl.h"
+#include "utils.h"
+
+static int separate_sysctl_key_value (char* sysctl_key_value, char** sysctl_key, char** sysctl_value);
+static int write_sysctl_to_file (char * sysctl_key, char* sysctl_value);
+const char *sysctl_delim = ":";
+
+int configure_sysctls (char * const sysctls) {
+  char* sysctl = strtok(sysctls, sysctl_delim);
+  char* key = NULL;
+  char* value = NULL;
+  while (sysctl) {
+    if (separate_sysctl_key_value (sysctl, &key, &value) < 0)
```

Comment:
We need to be consistent in the space between function call and arguments. I favor no spaces.

Reply:
I agree, but @giuseppe introduced the convention. I'll add all to have spaces, and the three of us can bicker about it when he's back :)